src/transform/FooterReadMore.js

```diff
@@ -169,29 +169,29 @@ const showReadMorePages = (pages, containerID, saveButtonClickHandler, titlesSho
  */
 const queryParameters = (title, count) => ({
   action: 'query',
-  continue: '',
-  exchars: 256,
-  exintro: 1,
-  exlimit: count,
-  explaintext: '',
   format: 'json',
+  formatversion: 2,
+  prop: 'extracts|pageimages',
+
+  // https://www.mediawiki.org/wiki/API:Search
+  // https://www.mediawiki.org/wiki/Help:CirrusSearch
   generator: 'search',
-  gsrinfo: '',
-  gsrlimit: count,
-  gsrnamespace: 0,
-  gsroffset: 0,
-  gsrprop: 'redirecttitle',
-  gsrsearch: `morelike:${title}`,
-  gsrwhat: 'text',
-  ns: 'ppprop',
-  pilimit: count,
-  piprop: 'thumbnail',
-  pithumbsize: 120,
-  prop: 'pageterms|pageimages|pageprops|revisions|extracts',
-  rrvlimit: 1,
-  rvprop: 'ids',
-  wbptterms: 'description',
```

Comment:
`wbptterms` is used to restrict `pageterms` to just the description. If omitted you'd get all of alias, description, label. See https://en.wikipedia.org/w/api.php?action=help&modules=query%2Bpageterms

If you only need the description then it's probably better to use `wbptterms=description`.